libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/filescan/ComponentPatternValidator.java

```diff
@@ -179,18 +181,13 @@ private static void processSharedIncludePatterns(Set<File> removedParentFiles, S
                             if (FileUtils.matches(sharedIncludePattern, FileUtils.normalizePathToLinux(file))) {
```

Comment:
optimize; move FileUtils.normalizePathToLinux(file) as variable out of the for loop